app/models/validations/financial_validations.rb

```diff
@@ -58,4 +58,76 @@ def validate_tshortfall(record)
       record.errors.add :tshortfall, I18n.t("validations.financial.hbrentshortfall.outstanding_no_benefits")
     end
   end
+
+  CHARGE_MAXIMUMS = {
```

Comment:
This constant can probably be private too? Not expected to be called from anywhere else right?